.goreleaser.yml

```diff
@@ -77,3 +80,19 @@ dockers:
     - "sonatypecommunity/nancy:{{ .Tag }}"
     - "sonatypecommunity/nancy:v{{ .Major }}"
     - "sonatypecommunity/nancy:v{{ .Major }}.{{ .Minor }}"
```

Comment:
If we are removing `v` from the archives should we also remove it here?? @DarthHater thoughts??

Reply:
I didn't see the issue here.

```
   • ARCHIVES         
      • skip archiving            binary=nancy.exe
      • skip archiving            binary=nancy
      • creating                  archive=dist/nancy-linux.386-vv0.1.5-next-2b79e57f21b0c370f22032c791da9c186f4c3da8.tar.gz
      • creating                  archive=dist/nancy-darwin.amd64-vv0.1.5-next-2b79e57f21b0c370f22032c791da9c186f4c3da8.tar.gz
      • skip archiving            binary=nancy
      • skip archiving            binary=nancy
      • creating                  archive=dist/nancy-darwin.386-vv0.1.5-next-2b79e57f21b0c370f22032c791da9c186f4c3da8.tar.gz
      • skip archiving            binary=nancy
      • creating                  archive=dist/nancy-windows.amd64-vv0.1.5-next-2b79e57f21b0c370f22032c791da9c186f4c3da8.zip
      • skip archiving            binary=nancy.exe
      • creating                  archive=dist/nancy-linux.amd64-vv0.1.5-next-2b79e57f21b0c370f22032c791da9c186f4c3da8.tar.gz
      • creating                  archive=dist/nancy-windows.386-vv0.1.5-next-2b79e57f21b0c370f22032c791da9c186f4c3da8.zip
```

Reply:
Cool I misunderstood what it was doing👍

Reply:
It seems to only do this in `--snapshot` mode. I've reverted that change.

Reply:
It was `{{ .Tag }}` vs `{{ .Version }}` in the snapshot section.